celery/utils/serialization.py

```diff
@@ -267,27 +260,14 @@ def jsonify(obj,
         return unknown_type_filter(obj)
 
 
-# Since PyPy 3 targets Python 3.2, 'raise exc from None' will
-# raise a TypeError so we need to look for Python 3.3 or newer
-if PY33:  # pragma: no cover
-    from vine.five import exec_
-    _raise_with_context = None  # for flake8
-    exec_("""def _raise_with_context(exc, ctx): raise exc from ctx""")
-
-    def raise_with_context(exc):
-        exc_info = sys.exc_info()
-        if not exc_info:
-            raise exc
-        elif exc_info[1] is exc:
-            raise
-        _raise_with_context(exc, exc_info[1])
-else:
-    def raise_with_context(exc):
-        exc_info = sys.exc_info()
-        if not exc_info:
-            raise exc
-        if exc_info[1] is exc:
-            raise
-        elif exc_info[2]:
-            reraise(type(exc), exc, exc_info[2])
+from vine.five import exec_
```

Comment:
I think this import needs to be on top now since it is not in a conditional branch?